config/logevent/logevent.go

```diff
@@ -126,12 +126,13 @@ func (t LogEvent) Get(field string) (v interface{}) {
 	switch field {
 	case "@timestamp":
 		v = t.Timestamp
-	case "message":
```

Comment:
You already handle `message` in SetValue(), so Get() should return from `message` property.